modules/ROOT/pages/intro-error-handlers.adoc

```diff
@@ -39,28 +37,37 @@ It’s important to note the UNKNOWN type, which is used when no clear reason fo
 
 When it comes to connectors, each connector defines its error type hierarchy considering the core runtime one, though CONNECTIVITY and RETRY_EXHAUSTED types are always present because they are common to all connectors.
 
+[[error_handlers]]
 == Error Handlers
 
 Mule 4 has redesigned error handling by introducing the `error-handler` component, which can contain any number of internal handlers and can route an error to the first one matching it. Such handlers are `on-error-continue` and `on-error-propagate`, which both support matching through an error type (or group of error types) or through an expression (for advanced use cases). These are quite similar to the Mule 3 choice (`choice-exception-strategy`), catch (`catch-exception-strategy`), and rollback (`rollback-exception-strategy`) exceptions strategies However, they are much simpler and more consistent.
 
 If an error is raised in Mule 4, an error handler will be executed and the error will be routed to the first matching handler. At this point, the error is available for inspection, so the handlers can execute and act accordingly, relative to the component where they are used (a Flow or Try scope):
 
-* An `on-error-continue` will execute and use the result of the execution, as the result of its owner (as if the owner had actually completed the execution successfully). Any transactions at this point would be committed as well.
-* An `on-error-propagate` will roll back any transactions, execute, and use that result to re-throw the existing error, meaning its owner will be considered as “failing.”
+* An `on-error-continue` executes and uses the result of the execution as the
+result of its owner (as though the owner completed the execution successfully).
+Any transactions at this point are committed, as well.
+* An `on-error-propagate` rolls back any transactions, executes, and uses that
+result to re-throw the existing error, meaning its owner is considered to be
+“failing.”
```

Comment:
```suggestion
“failing”.
```

Reply:
Americans are weird that way: https://www.thepunctuationguide.com/british-versus-american-style.html